docs/install/create-project.md

```diff
@@ -117,7 +117,7 @@ Use {term}`Corepack` to enable Yarn, which was already installed with the {ref}`
 1.  Open a terminal and type:
 
     ```shell
-    corepack enable
+    npm i -g corepack@latest && corepack enable
```

Comment:
Revert, since this method is deprecated. We're considering gutting this page. See https://github.com/plone/documentation/issues/1877#issuecomment-2692613756